docs/examples/questdb.md

```diff
@@ -0,0 +1,69 @@
+# QuestDB
+
+Not available until the next release of testcontainers-go <a href="https://github.com/testcontainers/testcontainers-go"><span class="tc-version">:material-tag: main</span></a>
+
+## Introduction
+
+The Testcontainers module for QuestDB.
+
+## Adding this module to your project dependencies
+
+Please run the following command to add the QuestDB module to your Go dependencies:
+
+```
+go get github.com/testcontainers/testcontainers-go/examples/questdb
+```
+
+## Usage example
+
+<!--codeinclude-->
+[Creating a QuestDB container](../../examples/questdb/questdb.go)
+<!--/codeinclude-->
+
+<!--codeinclude-->
+[Test for a QuestDB container](../../examples/questdb/questdb_test.go)
+<!--/codeinclude-->
+
+## Module reference
+
+The QuestDB module exposes one entrypoint function to create the QuestDB container, and this function receives two parameters:
+
+```golang
+func RunContainer(ctx context.Context, opts ...testcontainers.ContainerCustomizer) (*QuestDBContainer, error)
+```
```

Comment:
@mdelapenya ,
This part of documentation seems to be erronous. Shall modules and example both have an entrypoint with `RunContainer` ?

Reply:
I have a plan to generate an `example_test.go` file in every module, in which we only add Go examples code, which will be rendered in the markdown files (please see the Artemis module). An example module will not have real API, only examples.

Since we are not there yet, the example must have a private function. We could even make it simpler and name that function `runContainer` so we just have to lowercase the initial upper char.

Reply:
I see !